guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc

```diff
@@ -4,6 +4,12 @@
 
 This section describes how to upgrade {ProjectServer} from {ProjectVersionPrevious} to {ProjectVersion}.
 You can upgrade from any minor version of {ProjectName} Server {ProjectVersionPrevious}.
+However, to access the latest migration and tooling fixes, {Team} recommends that you upgrade to the latest minor version of {ProjectServer} before upgrading from {ProjectVersionPrevious} to {ProjectVersion}.
+
+Although the process of upgrading from {ProjectVersionPrevious} to {ProjectVersion} also migrates Pulp content, this can take some considerable time.
+You can reduce this time by following the procedure in xref:migrating_pulp_content[]
+
+If you have not pre-migrated content from Pulp 2 to Pulp 3, you must upgrade to the latest version of {ProjectServer} before starting to migrate Pulp content.
```

Comment:
this will also need to be wrapped in a ` ifdef::satellite[]`

Reply:
added